libs/wire-api/src/Wire/API/Routes/Internal/Brig/EJPD.hs

```diff
@@ -54,69 +50,131 @@ import Wire.Arbitrary (GenericUniform (..))
 newtype EJPDRequestBody = EJPDRequestBody {ejpdRequestBody :: [Handle]}
   deriving stock (Eq, Show, Generic)
   deriving (Arbitrary) via (GenericUniform EJPDRequestBody)
-  deriving (ToSchema) via CustomSwagger '[FieldLabelModifier (CamelToSnake, StripSuffix "_body")] EJPDRequestBody
+  deriving (OpenAPI.ToSchema) via S.CustomSwagger '[S.FieldLabelModifier (S.CamelToSnake, S.StripSuffix "_body")] EJPDRequestBody
 
 newtype EJPDResponseBody = EJPDResponseBody {ejpdResponseBody :: [EJPDResponseItem]}
   deriving stock (Eq, Show, Generic)
   deriving (Arbitrary) via (GenericUniform EJPDResponseBody)
-  deriving (ToSchema) via CustomSwagger '[FieldLabelModifier (CamelToSnake, StripSuffix "_body")] EJPDResponseBody
 
 data EJPDResponseItem = EJPDResponseItem
-  { ejpdResponseUserId :: UserId,
+  { ejpdResponseUserId :: Qualified UserId,
     ejpdResponseTeamId :: Maybe TeamId,
     ejpdResponseName :: Name,
     ejpdResponseHandle :: Maybe Handle,
     ejpdResponseEmail :: Maybe Email,
     ejpdResponsePhone :: Maybe Phone,
     ejpdResponsePushTokens :: Set Text, -- 'Wire.API.Push.V2.Token.Token', but that would produce an orphan instance.
-    ejpdResponseContacts :: Maybe (Set (Relation, EJPDResponseItem)),
+    ejpdResponseContacts :: Maybe (Set EJPDContact),
     ejpdResponseTeamContacts :: Maybe (Set EJPDResponseItem, NewListType),
-    ejpdResponseConversations :: Maybe (Set (Text, ConvId)), -- name, id
+    ejpdResponseConversations :: Maybe (Set EJPDConvInfo),
     ejpdResponseAssets :: Maybe (Set Text) -- urls pointing to s3 resources
   }
   deriving stock (Eq, Ord, Show, Generic)
   deriving (Arbitrary) via (GenericUniform EJPDResponseItem)
-  deriving (ToSchema) via CustomSwagger '[FieldLabelModifier CamelToSnake] EJPDResponseItem
 
-instance ToJSON EJPDRequestBody where
-  toJSON (EJPDRequestBody hs) = object ["ejpd_request" .= hs]
+data EJPDConvInfo = EJPDConvInfo {ejpdConvName :: Text, ejpdConvId :: Qualified ConvId}
+  deriving stock (Eq, Ord, Show, Generic)
+  deriving (Arbitrary) via (GenericUniform EJPDConvInfo)
+  deriving (Aeson.ToJSON, Aeson.FromJSON, OpenAPI.ToSchema) via Schema EJPDConvInfo
+
+instance ToSchema EJPDConvInfo where
+  schema =
+    object "EJPDConvInfo" $
+      EJPDConvInfo
+        <$> ejpdConvName .= field "conv_name" schema
+        <*> ejpdConvId .= field "conv_id" schema
+
+-- ejpdContactError :: FederationError,
+-- (^ we could include that, but we don't want to import wire-federation-api, or mess
+-- around with type arguments all over the place.)
```

Comment:
i think ormolu has move a comment again.  could you try to find a better spot where it sticks?